docs/user/masternodes/setup-testnet.rst

```diff
@@ -972,12 +1017,13 @@ Finishing up
 
 Ensure services managed by ``pm2`` start on reboot::
 
-  cat<<"EOF"|crontab
-  * * * * * cd ~/sentinel && ./bin/python bin/sentinel.py 2>&1 >> sentinel-cron.log
-  @reboot { sleep 5;cd ~/platform&&pm2 start yarn --name "drive" -- workspace @dashevo/drive abci;}
-  @reboot { sleep 6;cd ~/platform&&pm2 start yarn --name "dapi" -- workspace @dashevo/dapi api;}
-  @reboot { sleep 7;cd ~/platform&&pm2 start yarn --name "dapi" -- workspace @dashevo/dapi core-streams;}
-  EOF
+  pm2 save
+  pm2 startup
+
+Copy and paste the resulting command to ensure PM2 restarts processes on
+boot::
+
+  sudo env PATH=$PATH:/home/dash/.nvm/versions/node/v16.20.1/bin /home/dash/.nvm/versions/node/v16.20.1/lib/node_modules/pm2/bin/pm2 startup systemd -u dash --hp /home/dash
```

Comment:
The node version here may vary (for me it was 16.20.2) causing this line to fail

Reply:
Pinned Node version